holmes/core/supabase_dal.py

```diff
@@ -262,5 +266,64 @@ def get_workload_issues(self, resource: dict, since_hours: float) -> List[str]:
             return data
 
         except:
-            logging.exception("failed to fetch workload issues data")
+            logging.exception("failed to fetch workload issues data", exc_info=True)
+            return []
+
+    def upsert_holmes_status(self, holmes_status_data: dict) -> None:
+        updated_at = datetime.now().isoformat()
+        try:
+            res = (
+                self.client
+                .table(HOLMES_STATUS_TABLE)
+                .upsert({
+                    "account_id": self.account_id,
+                    "updated_at": updated_at,
```

Comment:
Unfortunately, it won't be solved that easily,  NOW() works only for INSERT, not UPDATE. So it would require from us to create trigger. I don't know what's better solution - trigger in our database or update relying on clients

Reply:
`"updated_at": "NOW()",` should work as expected.

Reply:
I suspect supabase just copies the column content into the SET part of a postgresql upsert.